### ISSUE TITLE
Class -> interface change doesn't detect changed class name or namespace

### ISSUE THREAD

**Issue by Majkl578** [Roave/BackwardCompatibilityCheck#98]
When class Foo\Test:
```php
<?php

namespace Foo;

class Test {}
```
becomes interface Foo\Bar\Test:
```diff
diff --git a/src/test.php b/src/test.php
index 20c1661..67214c4 100644
--- a/src/test.php
+++ b/src/test.php
@@ -1,5 +1,5 @@
 <?php
 
-namespace Foo;
+namespace Foo\Bar;
 
-class Test {}
+interface Test {}
```

Only class->interface is detected, not the changed namespace (applies to class name within same namespace too):
```
[BC] CHANGED: Class Foo\Test became an interface
1 backwards-incompatible changes detected
```

**Comment by Ocramius** [comment]
Is it not detected as removed class?

**Comment by Majkl578** [comment]
Nope.

**Comment by Ocramius** [comment]
Handled in #100
